src/QsCompiler/Compiler/FunctorGeneration.cs

```diff
@@ -9,7 +9,7 @@
 using Microsoft.Quantum.QsCompiler.ReservedKeywords;
 using Microsoft.Quantum.QsCompiler.SyntaxTokens;
 using Microsoft.Quantum.QsCompiler.SyntaxTree;
-
+using Microsoft.Quantum.QsCompiler.Transformations.CodeTransformationsNS;
```

Comment:
This seems like an odd namespace name? And it doesn't actually seem to be used anywhere?

Reply:
It is used throughout the file. Not sure why it is showing up as unused on your setup. Maybe rebuilding will fix it?

Reply:
This namespace has been dropped.